### PR TITLE
Describe how to load a C# DLC

### PR DESCRIPTION
**Pull request by xsellier**
Explains how to load a PCK made from a C# project.

### CODE REVIEW

File: getting_started/workflow/export/exporting_pcks.rst [hunk 1]
```diff
@@ -120,6 +120,11 @@ file in the directory of the games executable. The PCK file contains a
     subfolder). However, it is also a way of creating patches for one's own game. A
     PCK file of this kind can fix the content of a previously loaded PCK.
 
+.. note::
+    For a C# project you need to build the DLL and place it in the project directory first.
```
Comment (Calinou) [inline getting_started/workflow/export/exporting_pcks.rst]:
```suggestion
    For a C# project, you need to build the DLL and place it in the project directory first.
```

File: getting_started/workflow/export/exporting_pcks.rst [hunk 2]
```diff
@@ -120,6 +120,11 @@ file in the directory of the games executable. The PCK file contains a
...
+    Then, before loading the resource pack you need to load its DLL as following
```
Comment (Calinou) [inline getting_started/workflow/export/exporting_pcks.rst]:
```suggestion
    Then, before loading the resource pack, you need to load its DLL as follows:
```

**Comment by xsellier** [comment]
Thank you @Calinou

**Comment by Calinou** [comment]
Thanks!

**Pull request merged**